main.go

```diff
@@ -59,6 +45,38 @@ func update() {
 	fmt.Println("Updated Successfully.")
 }
 
+func parseJSON(s string) {
```

Comment:
Also give an example/add it to the readme on how to call this function from the binary. Its not straightforward to figure that out just by reading the code